Tools/autotest/common.py

```diff
@@ -8248,6 +8255,52 @@ def accelcal(self):
         if ex is not None:
             raise ex
 
+    def ahrstrim(self):
+        # setup with non-zero accel offsets
+        self.set_parameters({
+            "INS_ACCOFFS_X" : 0.7,
+            "INS_ACCOFFS_Y" : -0.3,
+            "INS_ACCOFFS_Z" : 1.8,
+            "INS_ACC2OFFS_X" : -2.1,
+            "INS_ACC2OFFS_Y" : 1.9,
+            "INS_ACC2OFFS_Z" : 2.3,
+            "SIM_ACC1_BIAS_X" : 0.7,
+            "SIM_ACC1_BIAS_Y" : -0.3,
+            "SIM_ACC1_BIAS_Z" : 1.8,
+            "SIM_ACC2_BIAS_X" : -2.1,
+            "SIM_ACC2_BIAS_Y" : 1.9,
+            "SIM_ACC2_BIAS_Z" : 2.3,
+            "AHRS_TRIM_X"    : 0.05,
+            "AHRS_TRIM_Y"    : -0.03,
+            "SIM_ACC_TRIM_X" : -0.04,
+            "SIM_ACC_TRIM_Y" : 0.05,
+            })
+        expected_parms = {
+            "AHRS_TRIM_X" : -0.04,
+            "AHRS_TRIM_Y" : 0.05,
+        }
+
+        self.progress("Starting ahrstrim")
+        self.mav.mav.command_long_send(self.sysid_thismav(), 1,
+                                    mavutil.mavlink.MAV_CMD_PREFLIGHT_CALIBRATION, 0,
+                                    0, 0, 0, 0, 2, 0, 0)
+                
+        self.mavproxy.expect("Trim OK", timeout=20)
```

Comment:
```suggestion
        self.context_collect('STATUSTEXT')
        self.run_cmd(mavutil.mavlink.MAV_CMD_PREFLIGHT_CALIBRATION, 0,
                                    0, 0, 0, 0, 2, 0, 0)
        self.wait_statustext('Trim OK', check_context=True)
        self.context_pop()                
```

Reply:
I undid this as it doesn't work for AHRS trim call